R/ml_estimation.R

```diff
@@ -423,7 +423,7 @@ print.ml_estimation <- function(x,
 
 
 
-#' Log-Likelihood Profile Function for Log-Location-Scale Distributions with Threshold
+#' Log-Likelihood Profile Function for Parametric Lifetime Distributions
```

Comment:
'with Threshold' at the end since this is the quantity for which the profile is done